main/src/library/Console.flix

```diff
@@ -93,18 +93,61 @@ mod Console {
         console() |> Option.forEach(consoleFlush)
 
     ///
-    /// Returns `s` formatted as bold.
+    /// Helper to apply the font style to the string `s` related to the terminal color supports
     ///
-    pub def bold(s: a): String with ToString[a] =
+    def applyFontStyle(s: a , style: (String, String)): String with ToString[a] =
```

Comment:
If this is a private function, it would usually come later in the file, after all its uses.